doc/api/dns.md

```diff
@@ -353,15 +353,15 @@ be an object with the following properties:
 * `minttl`
 
 ```js
-{
+({
```

Comment:
This doesn't feel right to me... If everybody else is okay with this, then fine.